test/modules/internet.spec.ts

```diff
@@ -421,6 +421,14 @@ describe('internet', () => {
           const username = faker.internet.userName('大羽', '陳');
           expect(username).includes('hlzp8d');
         });
+
+        it('should provide a fallback special unicode characters', () => {
+          const username = faker.internet.userName({
+            firstName: '🐼',
+            lastName: '❤️',
+          });
+          expect(username).includes('2qt8');
```

Comment:
Sorry to bother you again.
While this test is good, I don't think we will know what '2qt8' is supposed to mean in 2 weeks from now.
Can we give that value a descriptive name and/or even generate it from the hex representation of the emoji? The hex value is easier to look up and verify IMO.

 ```suggestion
          const pandaHexValue = 0x1F43C;
          // we represent unicode in base 36
          const expectPandaString = pandaHexValue.toString(36);
          expect(username).includes('2qt8');
```

Reply:
We don't do that for the chinese version either and IMO it doesn't matter either as it is some weird unicode -> ascii mapping anyway.